testing/test-utils/src/lib/utils/os-agnostic-paths.ts

```diff
@@ -4,8 +4,97 @@ import type {
   AuditReport,
 } from '@code-pushup/models';
 
-export function osAgnosticPath(path: string): string {
-  return path.replace(process.cwd(), '<CWD>').replace(/\\/g, '/');
+export const AGNOSTIC_PATH_SEP_REGEX = /[/\\]/g;
+export const OS_AGNOSTIC_PATH_SEP = '/';
+export const OS_AGNOSTIC_CWD = `<CWD>`;
```

Comment:
These constants seem like they should be internal.

```suggestion
const AGNOSTIC_PATH_SEP_REGEX = /[/\\]/g;
const OS_AGNOSTIC_PATH_SEP = '/';
const OS_AGNOSTIC_CWD = `<CWD>`;
```